python_modules/dagit/dagit/app.py

```diff
@@ -209,6 +212,7 @@ def index_view(*args, **kwargs):  # pylint: disable=unused-argument
                     rendered_template.replace('href="/', f'href="{app_path_prefix}/')
                     .replace('src="/', f'src="{app_path_prefix}/')
                     .replace("__PATH_PREFIX__", app_path_prefix)
+                    .replace("__TELEMETRY_ENABLED__", str(telemetry_enabled).lower())
```

Comment:
I think this needs to replace the quotes as well, e.g. `"\"__TELEMETRY_ENABLED__\""`, otherwise you could end up with

```
"telemetry_action": "false",
```

Reply:
I tested this in dagit and it seemed to work without the quotes.